solr/core/src/java/org/apache/solr/handler/tagger/XmlOffsetCorrector.java

```diff
@@ -47,19 +45,13 @@
 
   //TODO use StAX without hard requirement on woodstox.   xmlStreamReader.getLocation().getCharacterOffset()
 
-  private static final XMLInputFactory2 XML_INPUT_FACTORY;
+  private static final XMLInputFactory XML_INPUT_FACTORY;
   static {
     // note: similar code in Solr's EmptyEntityResolver
-    XML_INPUT_FACTORY = new WstxInputFactory();
-    XML_INPUT_FACTORY.setXMLResolver(new XMLResolver() {
-      @Override
-      public InputStream resolveEntity(String publicId, String systemId, String baseURI, String namespace) {
-        return ClosedInputStream.CLOSED_INPUT_STREAM;
-      }
-    });
+    XML_INPUT_FACTORY = XMLInputFactory2.newFactory();
+    XML_INPUT_FACTORY.setXMLResolver((publicId, systemId, baseURI, namespace) -> ClosedInputStream.CLOSED_INPUT_STREAM);
```

Comment:
This is a slight bit of tech debt; just use Solr's EmptyEntityResolver and then also remove the comment.  This code pre-dated inclusion of Solr; it used to be a side project.